### PR TITLE
Phaser 2.8.4, partially fixed audio on iOS

### PR DESCRIPTION
**Pull request by h223chen**
iOS requires touch to enable sound. Sound was split across the top and bottom canvas, requiring you to actually touch the top canvas before those sounds would play. 

Put all the sound in the bottom canvas.

### CODE REVIEW

**Comment by mattpk** [comment]
@bmwong @h223chen This may make the victory sound not play, since we set paused = true to pause the bottom canvas.

**Comment by h223chen** [comment]
@mattpk ok will double check and fix if needed, thanks 👍

File: index.html [hunk 1]
```diff
@@ -248,9 +221,16 @@
...
+                game.load.audio('bgm', 'assets/audio/bgm.mp3')                
+                game.load.audio('lightning', 'assets/audio/lightning.mp3')
+                game.load.audio('victory', 'assets/audio/victory.mp3')
+                game.load.audio('resolve', 'assets/audio/resolve.mp3')
+                game.load.audio('profit', 'assets/audio/profit.mp3')
+                game.load.audio('warning', 'assets/audio/warning.mp3')
+
                 game.load.spritesheet('vortex', 'assets/sprites/vortexSprite.png', 64, 64);
                 game.load.spritesheet('vortexRed', 'assets/sprites/vortexRed.png', 64, 64);
-                gameInfo.load.spritesheet('attack', 'assets/sprites/attack.png', 522, 522);
```
Comment (bmwong) [inline index.html]: Will the attack sprites still work if we load them into `game`? We're using them in `gameInfo` [here](https://github.com/h223chen/match3-hackathon/blob/c4abfa42944a92b4307d8ada80810b17a2b4300b/index.html#L1020) and [here](https://github.com/h223chen/match3-hackathon/blob/c4abfa42944a92b4307d8ada80810b17a2b4300b/index.html#L1041)

File: index.html [hunk 2]
```diff
@@ -248,9 +221,16 @@
...
+                game.load.spritesheet('attack', 'assets/sprites/attack.png', 522, 522);
```
Comment (bmwong) [inline index.html:233]: Will the attack sprites still work if we load them into `game`? We're using them in `gameInfo` [here](https://github.com/h223chen/match3-hackathon/blob/c4abfa42944a92b4307d8ada80810b17a2b4300b/index.html#L1020) and [here](https://github.com/h223chen/match3-hackathon/blob/c4abfa42944a92b4307d8ada80810b17a2b4300b/index.html#L1041)

Comment (h223chen) [reply index.html:233]: fixed this, will have PR up for changes

**Pull request merged**